pkg/apis/storage/validation/validation_test.go

```diff
@@ -1774,6 +1802,15 @@ func TestCSIDriverValidation(t *testing.T) {
 				PodInfoOnMount: nil,
 			},
 		},
+		{
```

Comment:
also test nil, even though we default it, just in case defaulting changes in the future